Tests/UnitTests/IdentityStateTests.swift

```diff
@@ -570,22 +570,18 @@ class IdentityStateTests: XCTestCase {
         // Verify Event type and source pairs, and the correct ordering
         // 1. Update consent event
         // 2. Reset identity request
-        XCTAssertEqual(2, dispatchedEvents.count)
-        XCTAssertEqual(EventType.edgeConsent, dispatchedEvents[0].type)
-        XCTAssertEqual(EventSource.updateConsent, dispatchedEvents[0].source)
-        XCTAssertEqual(EventType.edgeIdentity, dispatchedEvents[1].type)
-        XCTAssertEqual(EventSource.resetComplete, dispatchedEvents[1].source)
+        XCTAssertEqual(1, dispatchedEvents.count)
+//        XCTAssertEqual(EventType.edgeConsent, dispatchedEvents[0].type)
+//        XCTAssertEqual(EventSource.updateConsent, dispatchedEvents[0].source)
+        XCTAssertEqual(EventType.edgeIdentity, dispatchedEvents[0].type)
+        XCTAssertEqual(EventSource.resetComplete, dispatchedEvents[0].source)
```

Comment:
Can you update the comment and remove comment code?

Reply:
Comment has been updated and commented code removed